### ISSUE TITLE
Possible to disable "jump-to-anywhere" on C-leftclick (Mac)

### ISSUE THREAD

**Issue by thomas-noe** [SublimeText/LaTeXTools#786]
When I run ST3 (3114) with LaTeX tools, I am not able to pull up spelling suggestions via control leftclick on my mac. Without LaTeX tool installed this problem does not occur. When I examined the console log, I found that my left click was triggering a "jump to anywhere" command which (no surprisingly as the text was not a cite ref etc) failed. 
Thanks in advance for your assistance.

**Comment by r-stein** [comment]
This is the osx version of https://github.com/SublimeText/LaTeXTools/issues/784. I am on this and it will be fixed soon.
Until it is fixed you can create a file `Default (OSX).sublime-mousemap` in your User folder with the content:
``` js
[
    { "button": "button1", "count": 1, "modifiers": ["ctrl"], "press_command": "context_menu" }
]
```
This will overwrite the LaTeXTools mouse binding.

**Comment by thomas-noe** [comment]
Thanks for your help. I really appreciate it. Great program. However, inserting the file in my User directory does not work. However, it does change the console message to 
"Jump in tex file.
Cursor is not inside a command"
Best,

**Comment by thomas-noe** [comment]
Apologies. I wrote the mouse-map file to the wrong directory. You work around does work and does restore functionality to spell check suggest. Thanks.

**Comment by r-stein** [comment]
Fixed in `v3.9.3`
